src/lib/tabs/tab.ts

```diff
@@ -65,6 +65,8 @@ export class MatTab extends _MatTabMixinBase implements OnInit, CanDisable, OnCh
    * Will be cleared if `aria-label` is set at the same time.
    */
   @Input('aria-labelledby') ariaLabelledby: string;
+  
+  @Input('unique-name') uniqueName: string;
```

Comment:
This input should be documented so that consumers understand what this input is for.

Reply:
I have added an explanation for this input.